pkg/postgres/migrations/202501061613_reindexTestnetForStaterootChange/up.go

```diff
@@ -0,0 +1,40 @@
+package _202501061613_reindexTestnetForStaterootChange
+
+import (
+	"database/sql"
+	"github.com/Layr-Labs/sidecar/internal/config"
+	"gorm.io/gorm"
+)
+
+type Migration struct {
+}
+
+// Up
+// In response to the rewards-v2 audit, we needed to change how a couple of model state roots
+// were calculated. While this doesnt affect rewards generation, it does affect state roots that
+// have already been calculated and stored for preprod and testnet. To ensure that all state roots
+// are consistent, we need to prune back to the block where rewards-v2 was first deployed to allow
+// rewards to be recaclulated with the new state root calculation.
+func (m *Migration) Up(db *sql.DB, grm *gorm.DB, cfg *config.Config) error {
+	var blockNumber uint64
+	if cfg.Chain == config.Chain_Preprod {
+		blockNumber = 2919262
```

Comment:
We deployed Rewards v2 to Preprod at height `2871556`.

Reference:
1. https://github.com/Layr-Labs/eigenlayer-contracts-metadata/blob/master/deploys/preprod/2024-12-04-13-10-v0.5.1-rewardsv2/deploy.json#L52
2. https://holesky.etherscan.io/tx/0x7d9d32087bcd686123e68cd55a712fce5963af60ae2028a0d769747fe79a877a